cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -50,6 +53,10 @@ type node struct {
 	// E.g. secrets are soft-owned by a cluster via a naming convention, but without an explicit OwnerReference.
 	softOwners map[*node]empty
 
+	// moveLabel is set to true if the CRD of this object has the "move" label attached.
+	// This ensures the node is moved, regardless of its owner refs.
+	moveLabel bool
```

Comment:
```suggestion
	// forceMove ensures the node is moved, regardless of its owner refs.
	forceMove bool
```
`moveLabel` does not make sense anymore, given the label is at CRD level and not at object level, so I'm proposing to use a more generic `forceMove`